amr-wind/boundary_conditions/MassInflowOutflowBC.cpp

```diff
@@ -72,4 +72,4 @@ void MassInflowOutflowBC::operator()(
     }
 }
 
-} // namespace amr_wind
\ No newline at end of file
+} // namespace amr_wind
```

Comment:
spurious commit I think

Reply:
From what I understand, it is good practice to have a newline at the end of file. So I turned on automatically adding a newline in VScode and kept this change. Let me know if that is not correct and I can undo it.